AdyenCard/Components/GiftCardComponent/GiftCardDetails.swift

```diff
@@ -37,6 +40,21 @@ public struct GiftCardDetails: PaymentMethodDetails {
         self.encryptedCardNumber = encryptedCardNumber
         self.encryptedSecurityCode = encryptedSecurityCode
     }
+    
+    /// Initializes the gift card payment details.
+    ///
+    /// - Parameters:
+    ///   - paymentMethod: The used gift card payment method.
+    ///   - encryptedCard: The encrypted card .
+    public init(paymentMethod: GiftCardPaymentMethod, encryptedCard: EncryptedCard) throws {
+        guard let number = encryptedCard.number,
+              let securityCode = encryptedCard.securityCode else { throw GiftCardComponent.Error.cardEncryptionFailed }
```

Comment:
Minor:
This seems off, but probably will be fixed by "formatter" before release.

Reply:
formatter puts it this way actually

Reply:
Ok 🤔